synapseclient/annotations.py

```diff
@@ -116,8 +121,15 @@ def _annotation_value_list_element_type(annotation_values: typing.List):
     return object
 
 
-def is_submission_status_annotations(annotations):
-    """Tests if the given dictionary is in the form of annotations to submission status"""
+def is_submission_status_annotations(annotations: collections.abc.Mapping) -> bool:
+    """Tests if the given dictionary is in the form of annotations to submission status
+
+    Arguments:
```

Comment:
Nit:  I think `Args` is also acceptable.  I think we either support both `Arguments` and `Args` or just one of them.  If one of them, I leave it to you what you think we should aim to support.

Reply:
They are both supported by mkdocs. So I am ok with both being used interchangably